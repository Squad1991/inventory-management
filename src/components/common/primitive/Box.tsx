import { styled } from '@compiled/react';

const Box = styled.div({
  display: 'flex',
});

export default React.memo(Box);
