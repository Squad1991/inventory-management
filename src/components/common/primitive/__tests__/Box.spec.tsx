import { render } from '@testing-library/react';
import React from 'react';
import Box from '~src/components/common/primitive/Box';

describe('Box component', () => {
  it('should render correctly', () => {
    const { container } = render(<Box />);
    expect(container).toMatchSnapshot();
  });
});
