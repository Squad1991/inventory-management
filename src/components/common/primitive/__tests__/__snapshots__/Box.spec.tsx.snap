// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Box component > should render correctly 1`] = `
<div>
  <style
    data-cmpld="true"
  >
    ._1e0c1txw{display:flex}
  </style>
  <div
    class="_1e0c1txw"
  />
</div>
`;
